features/fixtures/rails3/app/config/initializers/bugsnag.rb

```diff
@@ -1,7 +1,7 @@
 Bugsnag.configure do |config|
   config.api_key = ENV["BUGSNAG_API_KEY"] || ENV["BUGSNAG_API_KEY"]
   config.endpoint = ENV["BUGSNAG_ENDPOINT"] || ENV["BUGSNAG_ENDPOINT"]
-  config.session_endpoint = ENV["BUGSNAG_ENDPOINT"] || ENV["BUGSNAG_ENDPOINT"]
+  config.session_endpoint = ENV["BUGSNAG_SESSION_ENDPOINT"] || ENV["BUGSNAG_SESSION_ENDPOINT"]
```

Comment:
Depends what you mean by "right" 😀 It works and has been this way for years (since https://github.com/bugsnag/bugsnag-ruby/commit/cbbb8ced6156e246f55468c34670ff25b9aa26ec) but there's obviously no point to being this way

I'll make a task to redo how environment variables work in these fixtures to fix this and stuff like `auto_capture_sessions`:

```ruby
config.auto_capture_sessions = ENV["BUGSNAG_AUTO_CAPTURE_SESSIONS"] == "true" unless ENV["USE_DEFAULT_AUTO_CAPTURE_SESSIONS"] == "true"
```